src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -214,8 +214,11 @@ internal class ConnectionManager : IConnectionManager {
 
         private bool IsValidLocalConnection(string name, string path) {
             try {
-                var info = new RInterpreterInfo(name, path);
-                return info.VerifyInstallation();
+                Uri uri;
+                if (Uri.TryCreate(path, UriKind.Absolute, out uri) && uri.IsFile) {
```

Comment:
It is called `IsValidLocalConnection` after all... It didn't work for remote connections anyways. Currently CM does not restore remote connection - which should probably be filed as a separate bug.